### ISSUE TITLE
JS/Java encoding-decoding incompabilities

### ISSUE THREAD

**Issue by macfol** [InteractiveAdvertisingBureau/iabtcf-java#96]
If I encode using below code:

```java
final Instant now = Instant.now();

final TCStringEncoder.Builder tcStrBuilder =  TCStringEncoder.newBuilder()
       .version(2)
       .created(now)
       .lastUpdated(now)
       .cmpId(CMP_ID)
       .cmpVersion(cpmV)
       .isServiceSpecific(true)
       .useNonStandardStacks(false)
       .publisherCC("DE")
        //THIS LEADS TO '=' at the end of string->
        .addCustomPurposesConsent(toBitSet(List.of(1)));

final String encoded = tcStrBuilder.encode();
```
the resulted String will be `COyqOSYOyqOSYB9ABAENAAAgAAAAAAAAAAYgAAAAAAAA.YAAAAAAAAWA=`

No if I go to [js demo page](http://iabtcf.com/#/decode) and decode above string it will result in error. Only after removing the `=` at the end the encoding would be possible.

This '=' gets when I add custom puprose as in above example

**Comment by srinivas81** [comment]
@macfol I have raised the issue as Issue # 94 and have provided the fix. Please let me know if I can close this issue as a duplicate issue? 
The = is for padding, the V2 spec doesn't clearly say if padding is allowed or not, but it is definitely not url/web safe, hence we have decided to not have any padding.

**Comment by macfol** [comment]
@srinivas81 
If it's gonna be fixed in #94 then feel free to close this one :)

**Comment by srinivas81** [comment]
It is fixed through issue #94 . Closing it.
